lib/passport-wsfed-saml2/utils.js

```diff
@@ -4,6 +4,24 @@ var SamlAssertionParserError = require('./errors/SamlAssertionParserError');
 var SamlResponseParserError = require('./errors/SamlResponseParserError');
 var WSFederationResultParserError = require('./errors/WSFederationResultParserError');
 
+CERT_START = "-----BEGIN CERTIFICATE-----\n";
```

Comment:
This has been declared global, please scope with `const`

Reply:
done